osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -180,6 +181,20 @@ private void updateIdleState(bool isIdle)
                 State = ButtonSystemState.Initial;
         }
 
+        protected override bool OnKeyDown(KeyDownEvent e)
+        {
+            if (State == ButtonSystemState.Initial)
+            {
+                if (buttonsTopLevel.Any(b => e.PressedKeys.Contains(b.TriggerKey)))
+                {
+                    logo.Click();
```

Comment:
missing a null check?
```suggestion
                    logo?.Click();
```

Reply:
Theoretically this should never occur, since it can only go null when switching out to another screen (at which point these hotkeys cannot be triggered due to `State`) but I guess it's fine to add for safety.